client/extensions/woocommerce/app/dashboard/setup-tasks.js

```diff
@@ -1,74 +1,91 @@
 /**
  * External dependencies
  */
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 import React, { Component, PropTypes } from 'react';
 
 /**
  * Internal dependencies
  */
+import {
+	areSetupChoicesLoading,
+	getOptedOutOfShippingSetup,
+	getOptedOutofTaxesSetup,
+	getTriedCustomizerDuringInitialSetup
+} from 'woocommerce/state/sites/setup-choices/selectors';
+import {
+	fetchSetupChoices,
+	setOptedOutOfShippingSetup,
+	setOptedOutOfTaxesSetup,
+	setTriedCustomizerDuringInitialSetup,
+} from 'woocommerce/state/sites/setup-choices/actions';
 import { getLink } from 'woocommerce/lib/nav-utils';
-import SetupFooter from './setup-footer';
-import SetupHeader from './setup-header';
 import SetupTask from './setup-task';
 
-class Setup extends Component {
+class SetupTasks extends Component {
 	static propTypes = {
-		onFinished: PropTypes.func,
 		site: PropTypes.shape( {
+			ID: PropTypes.number.isRequired,
 			slug: PropTypes.string.isRequired,
 		} ),
-		storeHasBeenCustomized: PropTypes.bool,
-		storeHasProducts: PropTypes.bool,
-		storePaymentsAreSetUp: PropTypes.bool,
-		storeShippingIsSetUp: PropTypes.bool,
-		storeTaxesAreSetUp: PropTypes.bool,
 	};
 
-	// TODO - replace with props mapped from state when this info becomes available in state
-	static defaultProps = {
-		storeHasBeenCustomized: true,
-		storeHasProducts: false,
-		storePaymentsAreSetUp: false,
-		storeShippingIsSetUp: false,
-		storeTaxesAreSetUp: false,
+	state = {
+		showShippingTask: this.props.loading || ! this.props.optedOutOfShippingSetup,
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15223/commits/5b766e2f1cd5fc11d3bc6f0dbd547fc198cde9bb